guide/daily-spending-wallet/settings.md

```diff
@@ -0,0 +1,213 @@
+    mobile = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile.jpg"
+    mobileRetina = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile@2x.jpg"
+    alt-text = "An oil painting of a big, clunky machine with a big bitcoin gear"
+    caption = "How AI thinks Matisse would have visualized bitcoin settings."
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+
+
+This is more of a summary page that bring together all the application settings that
+the user encounters throughout the app experience into a single, concise form.
+
+
+-->
+
+# Settings
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/overview.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/overview@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/overview-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "The overview screen provides an easy-to-understand grouping of settings."
+%}
+
+As you probably noticed in previous pages, various settings can be configured contextually throughout the app experience (like [hiding and unhiding the balance]({{ '/guide/daily-spending-wallet/privacy/#hiding-sensitive-information' | relative_url }})). All these options, and a few extra ones, are bundled in the settings area.
+
+While not the most frequently visited part of an application, it fills an important functional role by giving users precise control over their experience.
+
+</div>
+
+## General
+
+This sub-section includes various non-essential settings that are not covered in the other categories and let users adjust the application to their context and personal preferences.
+
+{% include image-gallery.html pages = page.images_general %}
+
+## Fees
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/fees.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/fees@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/fees-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "The fees screen provides a clear overview of all fees that may be charged."
+%}
+
+Paying fees for transactions is unavoidable and typically not something users appreciate. Being transparent and communicating them clearly in context (e.g. when [reviewing a payment]({{ '/guide/daily-spending-wallet/sending/#review--approval' | relative_url }}) before sending it) is crucial. Since contextual information is selective, this fee screen in settings is needed to provide a concise summary of all the types of fees that may be charged.
```

Comment:
Not appreciate sounds off, users expect fees in many situations

Reply:
Do you like paying fees? Expecting and appreciating are different.

Reply:
You know, it didn't bother me when I read it, but I can see why it's a confusing sounding word for Bosch.

The problem is that the word "appreciate" in English can have 2 meanings -- essentially, it can mean to _enjoy_ or it can mean to _recognize_ or _understand_ something. I do not appreciate paying fees, but I do appreciate the need for fees and the role they play in a transaction. Same word, different meaning in a different context.

@GBKS It's not a blocker for me, but I think if you swapped it out with the word "enjoy" that would be unambiguous and undebatable.

Reply:
Good point. Changed to `enjoy`.

Reply:
Interesting point, didn't know that. Is that a UK / US English thing? (us auzzies do UK english fyi)